docs/demos/TrafficManagerProfile/session-build-multi-clsuter-app/README.md

```diff
@@ -0,0 +1,221 @@
+# Deploying a Multi-Cluster Application using fleet APIs
+
+## Introduction 🚀
+In this session, we will walk through how to deploy a Nginx server seamlessly across multiple production clusters in **eastus2euap** while routing traffic intelligently using **Azure Traffic Manager**.
+
+![](build-multi-cluster-app.png)
+
+## Prerequisites
+Before we begin, ensure you have the following:
+- An **Azure Kubernetes Fleet Manager** set up with two members.

```

Comment:
Hi Zhiying! Just a nit here: would it be better if we say `a fleet` rather than `an Azure K8s Fleet Manager`?